packages/showcases/css/stories/components/badge/badge.stories.js

```diff
@@ -0,0 +1,66 @@
+import overview from './examples/overview.html';
+import _default from './examples/default.html';
+import brand from './examples/brand.html';
+import reversed from './examples/reversed.html';
+import accent from './examples/accent.html';
+
+export default {
+  title: 'Components / Badge ',
+  argTypes: { onchange: { action: 'write' } },
+  parameters: {
+    actions: {
+      handles: ['mouseenter', 'click', 'focusin', 'focusout'],
+    },
+    design: {
+      type: 'figma',
+      url:
+        'https://www.figma.com/file/zDZIyayUlr1yTWrsi7cFoo/Vtmn---Web?node-id=1450%3A8683',
+    },
+  },
+};
+
+export const Overview = () => overview;
+Overview.parameters = {
+  docs: {
+    source: {
+      code: overview,
+    },
+  },
+};
+
+export const Default = () => _default;
+Default.parameters = {
+  docs: {
+    source: {
+      code: _default,
+    },
+  },
+};
```

Comment:
Are default and brand the same? If yes, you can delete this "Default" story as I did on other components

Reply:
No they have different properties, the color for instance.